client/Pages/EventTypeDetails/View.elm

```diff
@@ -770,7 +770,16 @@ authTab appsInfoUrl usersInfoUrl teamsInfoUrl eventType =
         Just authorization ->
             div [ class "dc-card auth-tab" ]
                 [ div [ class "auth-tab__content" ]
-                    [ AccessEditor.viewReadOnly
+                    if isQueryOutput then
+                      [ warningMessage
+                            "This is the Authorization of the output event-type!"
+                            "Authorization of queries are not visible in Nakadi-UI"
```

Comment:
I think this begs the next question: where is that visible?  Can we have a link to the docs or something like that?